scripts/travis/install.sh

```diff
@@ -12,12 +12,9 @@ echo "Install Python dependencies"
 ./peep.sh install -r requirements/default.txt
 echo
 
-# Installing dependencies for smoke tests
-if [[ $TEST_SUITE == "smoke" ]]; then
-  virtualenv venv_smoketests
-  source venv_smoketests/bin/activate
-  pip install -r smoketests/requirements.txt
-  deactivate
+# Installing dependencies for UI tests
+if [[ $TEST_SUITE == "ui" ]]; then
+  ./peep.sh install -r requirements/test.txt
```

Comment:
The main reason we had a different virtualenv before was that the Input smoke tests, which we copied everything from, were incompatible with with Input's normal requirements. If we can make the set of requirements all work together, that's fine. And CI will make sure it stays that way.